src/app/+item-page/edit-item-page/item-relationships/edit-relationship-list/edit-relationship-list.component.html

```diff
@@ -1,15 +1,26 @@
-<h5>{{getRelationshipMessageKey() | async | translate}}</h5>
+<h5>
```

Comment:
Just a minor aesthetic comment... this section title is indented related to the content. It seems the content is unrelated with that title.

Reply:
I don't really see what you mean, it seems to me this indentation is correct?

Reply:
For better understanding, please check the image in my comment: https://github.com/DSpace/dspace-angular/pull/545#discussion_r459494904

`<h5>` has a different indentation than the content.